src/main/resources/public/ts/directives/card-list/card-list.directive.ts

```diff
@@ -137,6 +137,7 @@ function directive($parse: IParseService) {
                             let form: BoardForm = new BoardForm();
                             form.cardIds = vm.cardIds;
                             await boardsService.updateBoard(vm.cards[0].boardId, form);
+                            $parse($scope.vm.onMove())({});
```

Comment:
Si jamais ta méthode est optionnel, il existe une méthode angulare: isFunction
qui s'utilise comme ça:
```suggestion
                            if (isFunction($scope.vm.onMove)) $parse($scope.vm.onMove())({});
```

Reply:
c'est pas optionnel mais merci pour l'info !